config/kibana.yml

```diff
@@ -1,8 +1,10 @@
 # Kibana is served by a back end server. This setting specifies the port to use.
 # server.port: 5601
 
-# This setting specifies the IP address of the back end server.
-# server.host: "0.0.0.0"
+# Specifies the public host name to which the Kibana server will bind.
+# The default is 'localhost', which usually means remote machines will not be able to connect.
+# To allow connections from remote users, set this parameter to IP or the DNS-name of the machine that is running Kibana.
+# server.host: "localhost"
```

Comment:
You're going to hate me, but I'm still not totally happy with this description. Can we say

```
# Specifies the address to which the Kibana server will bind. IP addresses and host names are both valid values.
# The default is 'localhost', which usually means remote machines will not be able to connect.
# To allow connections from remote users, set this parameter to a non-loopback address.
```

I think this is more accurate and it borrows from the terminology the ES docs use (referring to loopback address) so it'll look familiar across products in the stack. Sorry I'm being super nitpicky, I just think it's important to document these options so their purpose is crystal clear.